README.md

```diff
@@ -7,3 +7,5 @@ The purpose of the Americana style is to:
 * Express the American experience through cartography
 * Challenge the status quo by showcasing innovation and invention
 * Promote collaboration and common purpose in the American mapping community
+
+Read the [Contributor's Guide](CONTRIBUTE.md) to get started.
```

Comment:
I might just put everything the the README, not sure there's enough to warrant 3 setup/instructions files yet. Two of them named the same thing. Or at least the stuff in `style/CONTRIBUTE.md`

Reply:
Consolidating the two at root level.  I envision we'll be doing custom .mbtiles stuff soon since I've already run into things I can't do out of the box even within the motorway feature.